shadow-cljs.edn

```diff
@@ -129,7 +129,8 @@
     ;; needed because we use deref in tests
     :static-fns    false
     :optimizations :simple
-    :infer-externs true}}
+    :infer-externs true
+    :warnings      {:fn-deprecated false}}}
```

Comment:
FYI: Deprecation warnings for shadow-cljs `test` target weren't being ignored. I disabled this because we don't need to see the warnings all the time when the code is recompiled in the test REPL.